### PR TITLE
Adapt to the new RMT API.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/dht11.toit [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
+import .driver_ as driver
 
 class Dht11 extends driver.Driver:
 
   /**
   Constructs an instance of the Dht11 driver.
   Uses RMT (ESP's Remote Control peripheral) to talk to the sensor. It allocates
-    two RMT channels with the given $rx_channel_num and $tx_channel_num numbers.
-    These RMT channels must be unused.
+    two RMT channels. If the $in_channel_id and $out_channel_id is provided, uses
```
Comment (kasperl) [inline src/dht11.toit:13]: are provided

File: src/dht22.toit [hunk 2]
```diff
@@ -4,20 +4,20 @@
...
+import .driver_ as driver
 
 class Dht22 extends driver.Driver:
 
   /**
   Constructs an instance of the Dht22 driver.
   Uses RMT (ESP's Remote Control peripheral) to talk to the sensor. It allocates
-    two RMT channels with the given $rx_channel_num and $tx_channel_num numbers.
-    These RMT channels must be unused.
+    two RMT channels. If the $in_channel_id and $out_channel_id is provided, uses
```
Comment (kasperl) [inline src/dht22.toit:14]: are provided

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/dht11.toit (hunk 1 above)
Comment (floitsch) [reply src/dht11.toit]: changed to "in_channel_id and/or ..."

File: src/dht22.toit (hunk 2 above)
Comment (floitsch) [reply src/dht22.toit]: ditto.

**Pull request merged**